WordPress/Classes/ViewRelated/Blog/Site Settings/SiteSettingsViewController+Swift.swift

```diff
@@ -335,8 +335,8 @@ extension SiteSettingsViewController {
             return ""
         }
 
-        let formatter = BloggingRemindersScheduleFormatter(schedule: scheduler.schedule(for: blog))
-        return formatter.shortIntervalDescription.string
+        let formatter = BloggingRemindersScheduleFormatter()
+        return formatter.shortIntervalDescription(for: scheduler.schedule(for: blog)).string
```

Comment:
@diegoreymendez I think this is the appropriate conflict resolution –it's the only one that compiles 😅 and it matches your changes from #16799– but could you confirm?

I'm asking because I was surprised by having a conflict here. Following the Git history, I can see #16781 went into `develop` recently (i.e.: after the code freeze) and that it does similar work to #16800. 

My guess is that #16800 is a refinement on the #16781 work, so this is the appropriate resolution. Let me know otherwise. Cheers!

Reply:
This looks correct to me!